### PR TITLE
Add functions working group teams to peribolos

### PR DESCRIPTION
**Pull request by lance**
Signed-off-by: Lance Ball <lball@redhat.com>

Seems to be required for: https://github.com/knative/community/pull/1032

### CODE REVIEW

File: peribolos/knative.yaml [hunk 1]
```diff
@@ -806,6 +806,18 @@ orgs:
...
+        privacy: closed
+      Func Writers:
+        description: Grants write access to function related repositories
+        privacy: closed
+        repos:
+          func: write
+        teams:
+          Func WG Leads:
+            description: The Working Group Leads for Functions
+            privacy: closed
```
[the remark this answers is not in the document]
Comment (lance) [reply peribolos/knative.yaml]: > it's really annoying

Yes! Yes it is! 😭

**Comment by lance** [comment]
I am so confused.

**Comment by vaikas** [comment]
/test pull-knative-peribolos
Just in case this was fixed by the #1095 as well :)

**Comment by kvmware** [comment]
`pull-knative-peribolos` should be optional (can be merged even if it fails) now, while we get some answers from people working on peribolos.

**Pull request merged**